### PR TITLE
adding a test vector where did reso succeeds

### PR DESCRIPTION
**Pull request by jiyoontbd**
## Important!

Make sure that if you are updating a test vector that you test in the following repos before merging!

- [ ] https://github.com/TBD54566975/web5-go
- [ ] https://github.com/TBD54566975/web5-js
- [ ] https://github.com/TBD54566975/web5-kt
- [ ] https://github.com/TBD54566975/web5-swift
- [ ] https://github.com/TBD54566975/web5-dart

Clone each repo locally, make sure that each web5-spec submodule is pointed to your branch, and then run tests to make sure it is still passing in all repos!

### CODE REVIEW

File: test-vectors/did_dht/resolve.json [hunk 1]
```diff
@@ -1,6 +1,36 @@
 {
   "description": "did:dht resolution",
   "vectors": [
+    {
+      "description": "resolves successfully",
```
Comment (decentralgabe) [inline test-vectors/did_dht/resolve.json:5]: to be clear, is this expected to hit a DHT resolver? or resolve locally?

Comment (jiyoontbd) [reply test-vectors/did_dht/resolve.json:5]: i would like it to hit a did resolver. this pr is related to https://github.com/TBD54566975/web5-go/pull/156 where i found that web5-go's did dht resolution fails this test.

Comment (jiyoontbd) [reply test-vectors/did_dht/resolve.json:5]: ah, i remember web5-js we mocked out all did dht calls. so yea, maybe it should resolve locally. either way, we need a test vector where one resolves successfully - current test cases are all failure ones from what i recall

i will turn this PR back to draft state and make sure web5-go works.

Comment (decentralgabe) [reply test-vectors/did_dht/resolve.json:5]: fine if it hits a resolver, but it should not rely on a published DID. it should be a self contained test

**Pull request closed without merging**